From a75c60cedfd4fc0fbc0121a69eea57a6d33f07a6 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:48:58 -0400
Subject: [tools] perf/timechart: Remove some needless struct forward declarations

Message-id: <1408953210-26343-51-git-send-email-jolsa@redhat.com>
Patchwork-id: 89043
O-Subject: [PATCH RHEL7.1 BZ1133083 050/322] perf timechart: Remove some needless struct forward declarations
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 6e53ad9ff65226f79d5888452992222284c7e4d4
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Wed Nov 27 16:29:50 2013 -0300
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-timechart.c b/tools/perf/builtin-timechart.c
index 436cb5f..c246f02 100644
--- a/tools/perf/builtin-timechart.c
+++ b/tools/perf/builtin-timechart.c
@@ -55,14 +55,8 @@ static bool  tasks_only;
 static bool  with_backtrace;
 
 
-struct per_pid;
 struct per_pidcomm;
-
 struct cpu_sample;
-struct power_event;
-struct wake_event;
-
-struct sample_wrapper;
 
 /*
  * Datastructure layout:
@@ -155,7 +149,6 @@ struct wake_event {
 static struct power_event    *power_events;
 static struct wake_event     *wake_events;
 
-struct process_filter;
 struct process_filter {
  char   *name;
  int   pid;
-- 
1.7.1